mountpoint-s3-client/tests/put_object.rs

```diff
@@ -227,7 +230,48 @@ async fn test_put_checksums() {
 
     assert_eq!(checksums.len(), expected_checksums.len());
     for (checksum, expected_checksum) in checksums.into_iter().zip(expected_checksums.into_iter()) {
-        let encoded = Base64::encode_string(&expected_checksum.value().to_be_bytes());
+        let encoded = expected_checksum.to_base64();
         assert_eq!(checksum, encoded);
     }
 }
+
+#[test_case(true; "pass review")]
+#[test_case(false; "fail review")]
+#[tokio::test]
+async fn test_put_review(pass_review: bool) {
```

Comment:
This test should also check that `get_object` works if `true` and doesn't if `false`? And perhaps that the MPU was in fact aborted in the false case.